tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -217,20 +219,13 @@ describe('Target aggregates', () => {
       moment().date(1).add(1, 'month').format('YYYY-MM'),
     ];
```

Comment:
Can you use the factories for this test to create users, contacts, persons, and places? I used it in `/tests/e2e/default/contacts/edit.wdio-spec.js`, and it makes the code cleaner.